modules/openapi-generator/src/main/resources/python-legacy/api_client.mustache

```diff
@@ -552,7 +552,7 @@ class ApiClient(object):
         :return: Content-Type (e.g. application/json).
         """
         if not content_types:
-            return 'application/json'
```

Comment:
I am concerned that this could be a breaking change for servers/client that depended upon this functionality

Reply:
> I am concerned that this could be a breaking change for servers/client that depended upon this functionality.
How about adding a generator argument useJsonAsDefaultContentType and default it to True.

We've done something similar in other generators (e.g. Ruby) to remove such default behaviour (which was added before I worked on this project). No complains so far.